tests/test_pillow_simd.py

```diff
@@ -1,7 +1,14 @@
 # Copyright 2021 MosaicML. All Rights Reserved.
 
+import platform
+
 import PIL
+import pytest
 
 
+@pytest.mark.skipif(
+    'Linux' not in platform.system(),
+    reason="Pillow-SIMD not available on non-Linux systems.",
+)
```

Comment:
I have it on my intel Mac, so technically this comment isn't quite correct; but it's hard enough to install that I don't mind skipping the test.